yucca/image_processing/transforms/Gamma.py

```diff
@@ -11,44 +10,34 @@ def augment_gamma(
     invert_image=False,
     epsilon=1e-7,
     per_channel=False,
-    retain_stats: Union[bool, Callable[[], bool]] = False,
+    clip_to_input_range=False,
 ):
     if invert_image:
         data_sample = -data_sample
 
     if not per_channel:
-        retain_stats_here = retain_stats() if callable(retain_stats) else retain_stats
-        if retain_stats_here:
-            mn = data_sample.mean()
-            sd = data_sample.std()
         if np.random.random() < 0.5 and gamma_range[0] < 1:
             gamma = np.random.uniform(gamma_range[0], 1)
         else:
             gamma = np.random.uniform(max(gamma_range[0], 1), gamma_range[1])
-        minm = data_sample.min()
-        rnge = data_sample.max() - minm
-        data_sample = np.power(((data_sample - minm) / float(rnge + epsilon)), gamma) * rnge + minm
-        if retain_stats_here:
-            data_sample = data_sample - data_sample.mean()
-            data_sample = data_sample / (data_sample.std() + 1e-8) * sd
-            data_sample = data_sample + mn
+        mn = data_sample.min()
+        mx = data_sample.max()
+        rnge = mx - mn
+        data_sample = np.power(((data_sample - mn) / float(rnge + epsilon)), gamma) * rnge + mn
+        if clip_to_input_range:
+            data_sample = np.clip(data_sample, a_min=mn, a_max=mx)
     else:
         for c in range(data_sample.shape[0]):
-            retain_stats_here = retain_stats() if callable(retain_stats) else retain_stats
-            if retain_stats_here:
-                mn = data_sample[c].mean()
-                sd = data_sample[c].std()
             if np.random.random() < 0.5 and gamma_range[0] < 1:
                 gamma = np.random.uniform(gamma_range[0], 1)
             else:
                 gamma = np.random.uniform(max(gamma_range[0], 1), gamma_range[1])
-            minm = data_sample[c].min()
-            rnge = data_sample[c].max() - minm
-            data_sample[c] = np.power(((data_sample[c] - minm) / float(rnge + epsilon)), gamma) * float(rnge + epsilon) + minm
-            if retain_stats_here:
-                data_sample[c] = data_sample[c] - data_sample[c].mean()
-                data_sample[c] = data_sample[c] / (data_sample[c].std() + 1e-8) * sd
-                data_sample[c] = data_sample[c] + mn
+            mn = data_sample[c].min()
+            mx = data_sample[c].max()
+            rnge = mx - mn
+            data_sample[c] = np.power(((data_sample[c] - mn) / float(rnge + epsilon)), gamma) * float(rnge + epsilon) + mn
```

Comment:
Possible bug: In the original code `mn` was mean, now it is `min`. I am not sure that should change? Also, this is a really good example of why the shortening is not worth it.